.github/workflows/changelog.yml

```diff
@@ -69,3 +69,23 @@ jobs:
           key: ${{ runner.os }}-go-pkg-mod-${{ hashFiles('go.sum') }}
       - run: cd tools && go install github.com/client9/misspell/cmd/misspell
       - run: misspell -error -source text CHANGELOG.md
+
+name: Generate CHANGELOG
+on:
+  pull_request:
+    types: [closed]
+jobs:
```

Comment:
I think these top-level YAML attributes may override the original ones above instead of being additive.